client/src/components/MenuBar.css

```diff
@@ -27,3 +27,7 @@
 		letter-spacing: 0.6px;
 	}
 }
+.donateColor {
+	background-color: #14d547;
+	border-radius: 5px;
```

Comment:
Looks good, using react bootstrap colors would work as well but this is acceptable https://getbootstrap.com/docs/4.0/utilities/colors/